web/src/components/apps/AppVersionHistory.tsx

```diff
@@ -2083,6 +2149,30 @@ class AppVersionHistory extends Component<Props, State> {
                             )}
                           </div>
                         )}
+
+                      {/* not gitops, is embedded */}
+                      {!gitopsIsConnected &&
```

Comment:
you don't need to check for `gitopsIsConnected`. you can just check if it's an embedded cluster.